README.md

```diff
@@ -1,12 +1,42 @@
-# Auto Join Google Meet
-
-## Description
+# Auto Join Chrome Extension
 
 We are all fed up of keeping track of google meet links for our lectures. This chrome extension tries to make your life a bit easy!
+You just have to select the slot and the course and this extension opens the link at the given time ( you can customize the start time ) and also lets you auto join it!
+
+<b>Link : <b/>[Auto Join Chrome Extension](https://chrome.google.com/webstore/detail/auto-join-chrome-extensio/djlpjopjnkojimgiaefgbenbbkkknnln?hl=en)
```

Comment:
You should use an icon for the chrome store link
https://www.google.com/search?q=download+on+chrome+store+icon&newwindow=1&sxsrf=ALeKk00ILqRBq3_yl-RGQqN0zORMI2YC_g:1601016924379&source=lnms&tbm=isch&sa=X&ved=2ahUKEwiN0rno3IPsAhXQeisKHZaqDgsQ_AUoAXoECA4QAw&biw=1396&bih=766

Reply:
So instead of writing Auto Join Chrome Extension, I'll just add the icon?

Reply:
You can maybe just refer how other chrome extension repos do it.

Reply:
# [Auto Join Chrome Extension](https://chrome.google.com/webstore/detail/auto-join-chrome-extensio/djlpjopjnkojimgiaefgbenbbkkknnln?hl=en) ![output-onlinepngtools (1)](https://user-images.githubusercontent.com/56914284/94237918-31108980-ff2d-11ea-9133-1a3718e1b2a5.png)

Changed the heading to this. Is it okay?

Reply:
umm, not looking good.
Perhaps use the one which says "Available in the chrome store"
And we can make the picture the URL itself
No need for the text before it

Reply:
@anshulahuja98  done! Please have a look

Reply:
LGTM